app/components/browse/prefix_selector.rb

```diff
@@ -2,24 +2,48 @@
 
 module Browse
   class PrefixSelector < ViewComponent::Base
+    include Paths
+
     def prefix
       params[:prefix]
     end
 
     def clear_prefix_path
-      browse_subjects_path(merge_params)
+      browse_path(merge_params)
     end
 
     def prefix_path(letter)
-      browse_subjects_path(merge_params(prefix: letter))
+      browse_path(merge_params(prefix: letter))
     end
 
     def form_url
-      browse_subjects_url
+      raise ArgumentError, "no form URL defined for '#{action}'" unless respond_to?("#{action}_form_url", true)
+
+      send("#{action}_form_url")
```

Comment:
If we're doing this right, this can be removed and replaced with Paths#browse_url